### PR TITLE
Second pull request

### PR DESCRIPTION
**Pull request by Zven94**
My Portfolio
-----------------------------

In this pull I have add:

- in html: added `form` and `button` to sumit
- in style: added style for the `form` & `button section`

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -474,4 +474,117 @@
...
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    color: white;
+    background-color: #ff6b00;
+    padding: 10px 12px;
+    width: 131px;
+    height: 48px;
+    border: 1px solid #ff6b00;
+  }
```
Comment (Tufahel) [inline style.css:553]: - [ ] Kindly add `hover` for the `Get in Touch` button as it is important to match the given design template.

File: index.html [hunk 2]
```diff
@@ -234,10 +235,26 @@ <h6 class="h6">API Desing</h6>
...
-    <footer>
-    </footer>
+        <!-- Added form on the last page-->
+        <section class="section-11">
+                <h5 class="h5-2">I'm always interested in hearing 
+                about new projects, so if you'd like to chat 
+                please get in touch.</h5>
+                <form action="https://formspree.io/f/xayzvjyw" method="post" class="form">
+                    <input type="text" name="full_name" id="name" placeholder="Full name" required
+                    minlength="30">
```
Comment (Tufahel) [inline index.html:245]: - [ ] You have used `minlength="30"` here. But in the requirements, I can see it says, "Name field accepts `up to 30 chars.`" `Up to` means `Maximum`. So kindly replace with `maxlength="30"` to fix this issue.
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Zven94 :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Professional pull request
✔️ Correct git-flow
✔️ Linter checks are passing

## Required Changes ♻️

- [ ] I can see the setup info in your `README.MD` file is not updated.  This is a `Portfolio` project but you have added the clone link of `Hello Microverse`. Kindly update the info as per your project as it is professional to document your work perfectly.
- image


***Please kindly check the comments below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Zven94) [reply style.css:553]: Hi, how are you? Thank you for your time and your advice.
I noticed that i already have the hover in the program. Did I do it wrong?

Comment (Zven94) [reply style.css:553]: Oh I just solved it!. Thank you!

**Review by abdallahmalima — APPROVED** [review]
Hi @Zven94 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- form submit to formspree service ✔
- input fields have the correct `type` attribute ✔
- correct use of display flex

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**